### PR TITLE
feat(releases_test): finish implementing releases test

### LINKED ISSUE #90: finish the releases tests
`releases_test.go` needs some love, with particular attention to:
```
		// 500's everytime
		XIt("can deploy the app", func() {
			sess, err := start("deis pull deis/example-go -a %s", appName)
			Expect(err).To(BeNil())
			Eventually(sess, (10 * time.Minute)).Should(Exit(0))
			Eventually(sess).Should(Say("Creating build... done"))
		})
```
In addition, the tests context currently claims `Context("with a deployed app"` but the app isn't actually being deployed in the `BeforeEach` step

### PR DESCRIPTION
**Pull request by vdice**
Closes #90

### CODE REVIEW

File: tests/releases_test.go [hunk 1]
```diff
@@ -10,46 +10,51 @@ import (
...
 			Expect(err).To(BeNil())
 			Eventually(sess, (1 * time.Minute)).Should(Exit(0))
-			Eventually(sess).Should(Say("=== %s Releases", appName))
+			Eventually(sess).Should(Say("=== %s Releases", testApp.Name))
 			Eventually(sess).Should(Say(`v1\s+.*\s+%s created initial release`, testUser))
-		})
 
-		It("can rollback to a previous release", func() {
-			sess, err := start("deis releases:rollback v1 -a %s", appName)
+			// "can rollback to a previous release"
```
Comment (sgoings) [inline tests/releases_test.go]: (nit): do you want to vertically line up the text after the comment or the beginning of the comment?

**Comment by mboersma** [comment]
Passes here:
```console
Ran 1 of 89 Specs in 23.191 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 88 Skipped PASS | FOCUSED
```

File: tests/releases_test.go (hunk 1 above)
Comment (vdice) [inline tests/releases_test.go]: Bizarre, chalking this up to github... code appears correctly aligned in atom.  (re-spacing to proper alignment shows no diff)

**Comment by bacongobbler** [comment]
LGTM3 (code review)

File: tests/releases_test.go (hunk 1 above)
Comment (sgoings) [inline tests/releases_test.go]: :+1: looks good in an editor and there's no space/tab discrepancy. :ship:

**Pull request merged**